microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md

```diff
@@ -41,7 +41,7 @@ Command Palette provides access to Microsoft Edge commands, including various De
 
 To open Command Palette:
 
-1. In Microsoft Edge, press `Ctrl`+`Shift`+`Space`.  Command Palette opens.
+1. In Microsoft Edge, press `Ctrl`+`Q`.  Command Palette opens.
```

Comment:
Stable 105 Win: Ctrl+Shift+Space works.  Ctrl+Q does nothing.
Canary v__ Win: tbd

Stable 105 Mac: tbd
Canary v__ Mac: tbd

Reply:
The keyboard shortcut change is happening right now. Maybe you just haven't got the new code yet? @erdraud when will the new shortcut be available to test in Edge?

Reply:
@captainbrosset - Per erdraud, the Ctrl+Q shortcut key will be in Canary the week of September 19, 2022. The experiment (in flags page) is the old shortcut. The rollout (not flags page) is Ctrl+Q.

Reply:
@captainbrosset we have a PR on standby

Reply:
@captainbrosset - Per erdraud, The Ctrl+Q shortcut will be in by 108, possibly in as early as a later minor release of 107, but not yet in current minor release of 107.  2 changes: The displayed text in the flags page, and the behavior.